service/telemetry/logger.go

```diff
@@ -93,3 +91,51 @@ func newSampledCore(core zapcore.Core, sc *LogsSamplingConfig) zapcore.Core {
 		sc.Thereafter,
 	)
 }
+
+// minsevLoggerProvider wraps a LoggerProvider, and filters logs based on the
```

Comment:
I've opened https://github.com/open-telemetry/opentelemetry-go-contrib/issues/7549 to improve on this